conseil-common/src/main/scala/tech/cryptonomic/conseil/common/tezos/Fork.scala

```diff
@@ -0,0 +1,15 @@
+package tech.cryptonomic.conseil.common.tezos
+
+object Fork {
```

Comment:
Curious why this is an object by itself. In the future, what else might it contain other than `mainForkId`?

Reply:
I didn't have a proper place to put that constant into.
I did go for a separate object because it's unrelated to other generic tezos operations, essentially.

Also using it at call site explicitly hint at what we're talking about. That is, it's related to fork handling.